packages/scripts.nix

```diff
@@ -16,6 +16,83 @@
     text = builtins.readFile ./destroy-coco-aks.sh;
   };
 
+  upload-image = writeShellApplication {
+    name = "upload-image";
+    runtimeInputs = with pkgs; [
+      azure-cli
+      gnused
+      uplosi
+    ];
+    text =
+      let
+        image = pkgs.image-podvm;
+      in
+      ''
+        set -euo pipefail
+
+        subscriptionId=""
+        location="GermanyWestCentral"
+        resourceGroup=""
+
+        for i in "$@"; do
+          case $i in
+          --subscription-id=*)
+            subscriptionId="''${i#*=}"
+            shift
+            ;;
+          --location=*)
+            location="''${i#*=}"
+            shift
+            ;;
+          --resource-group=*)
+            resourceGroup="''${i#*=}"
+            shift
+            ;;
+          *)
+            echo "Unknown option $i"
+            exit 1
+            ;;
+          esac
+        done
+
+        set -x
+
+        # Create the resource group.
+        az group create \
+          --name "''${resourceGroup}" \
+          --location "''${location}"
+
+        # Create an uplosi config.
+        cat <<EOF > uplosi.conf
+        [base]
+        imageVersion = "0.0.1"
+        name = "contrast"
+        provider = "azure"
+
+        [base.azure]
+        subscriptionID = "''${subscriptionId}"
+        location = "''${location}"
+        resourceGroup = "''${resourceGroup}"
+        sharedImageGallery = "contrast"
```

Comment:
Apparently Azure doesn't allow multiple galleries with the same name even if they are in different resource groups:

![image](https://github.com/user-attachments/assets/db1ecb97-a497-4583-8fa9-e7f167156cb5)